themes/bootstrap3/templates/RecordTab/extendedholdingsils.phtml

```diff
@@ -0,0 +1,84 @@
+        <td colspan="2" class="holding-<?=$this->holding['availability'] ? "available" : "unavailable" ?>">
+          <div class="holding-details">
+            <span class="holding-field copy-number">
+              <?=$this->transEsc("Copy")?> <?=$this->escapeHtml($this->holding['number'])?>
+            </span>
+            <span class="holding-field barcode">
+              <?=$this->escapeHtml($this->holding['barcode'])?>
+            </span>
+            <span class="holding-field availability">
+              <? if ($this->holding['reserve'] == "Y"): ?>
+                <link property="availability" href="http://schema.org/InStoreOnly" />
+                <?=$this->transEsc("On Reserve - Ask at Circulation Desk")?><br />
+              <? endif; ?>
+  
+              <? if (isset($this->holding['use_unknown_message']) && $this->holding['use_unknown_message']): ?>
+                <span class="text-muted"><?=$this->transEsc("status_unknown_message")?></span>
+              <? else: ?>
+                <? if ($this->holding['availability']): ?>
+                  <? /* Begin Available Items (Holds) */ ?>
+                  <span class="text-success"><?=$this->transEsc("Available")?><link property="availability" href="http://schema.org/InStock" /></span>
+                <? else: ?>
+                  <? /* Begin Unavailable Items (Recalls) */ ?>
+                  <span class="text-danger"><?=$this->transEsc($this->holding['status'])?><link property="availability" href="http://schema.org/OutOfStock" /></span>
+                  <? if (isset($this->holding['returnDate']) && $this->holding['returnDate']): ?>
+                    <span class="small return-date"><?=$this->escapeHtml($this->holding['returnDate'])?></span>
+                  <? endif; ?>
+                  <? if (isset($this->holding['duedate']) && $this->holding['duedate']): ?>
+                    <span class="small due-date"><?=$this->transEsc("Due")?>: <?=$this->escapeHtml($this->holding['duedate'])?></span>
+                  <? endif; ?>
+                  <? if (isset($this->holding['requests_placed']) && $this->holding['requests_placed'] > 0): ?>
+                    <span class="request-count"><?=$this->transEsc("Requests")?>: <?=$this->escapeHtml($this->holding['requests_placed'])?></span>
+                  <? endif; ?>
+                <? endif; ?>
+              <? endif; ?>
+            </span>
+            <span class="holding-field place-hold">
+              <? if (isset($this->holding['link']) && $this->holding['link']): ?>
+                <a class="<?=$check ? 'checkRequest ' : ''?>placehold" <? if (!empty($this->holding['linkLightbox'])): ?>data-lightbox <? endif; ?>href="<?=$this->recordLink()->getRequestUrl($this->holding['link'])?>"><i class="fa fa-flag" aria-hidden="true"></i>&nbsp;<?=$this->transEsc($check ? "Check Hold" : "Place a Hold")?></a>
```

Comment:
These check variables don't seem to be initialized. You can [copy the setup from holdingsils](https://github.com/vufind-org/vufind/blob/master/themes/bootstrap3/templates/RecordTab/holdingsils.phtml#L92) replacing `$row` with `$this->holding`.

Reply:
```php
<?
  // AJAX Check record?
  $check = isset($this->holding['check']) && $this->holding['check'];
  $checkStorageRetrievalRequest = isset($this->holding['checkStorageRetrievalRequest']) && $this->holding['checkStorageRetrievalRequest'];
  $checkILLRequest = isset($this->holding['checkILLRequest']) && $this->holding['checkILLRequest'];
?>
```

Reply:
Perfect.